ib_insync/ib.py

```diff
@@ -1715,6 +1715,8 @@ async def qualifyContractsAsync(self, *contracts: Contract) -> \
                 self._logger.error(
                     f'Ambiguous contract: {contract}, '
                     f'possibles are {possibles}')
+                # return all possible contracts that match
```

Comment:
If we don't like returning the list of possibles by default i can also add an input flag if y'all want.

Reply:
Ok but what if you want this?
Sorry I'm not entirely following why this returns the valid output i'm after but for some reason it's not available to use?

> use reqContractDetails with a "wildcard" contract.

I am using this ep underneath, but i don't see how doing a qualify on an ambiguous should be less correct then a "request"? What exactly is the semantic different between these 2 requests and why would you use one over the other?

For example, [in the api docs](https://interactivebrokers.github.io/tws-api/basic_contracts.html#fut) they show how to request details using the `.SecType = "FUT+CONTFUT"` key:

> The security type "FUT+CONTFUT" can be used to request contract details about the futures and continuous futures on an underlying. This security type cannot be used with other functionality.

Reply:
I think part of my issue is i'm using the [stock search](https://interactivebrokers.github.io/tws-api/matching_symbols.html) system and it returns an index result for many equity futures contracts.

I haven't dug in yet but what's the motivation for using `.reqContractDetails()` over qualify?

Reply:
Ah ok will give it a shot and report back.

Reply:
@erdewit yeah was finally just trying this and I don't understand how it's supposed to work.

What is this ""wildcard" contract." that I would pass to `.reqContractDetails()`?

Reply:
Nm figured it out 😂 

Thanks for the tip, this method works just as well.